### PR TITLE
added user rake task set_admin_role

### LINKED ISSUE #5515: [v3] No practical way to create/use (super) admin accounts with SSO enabled.
After enabling OpenID Connect there is no obvious way to login with a local account (e.g. the local admin account).

### PR DESCRIPTION
**Pull request by SilentFlameCR**
Fixes #5515

### CODE REVIEW

File: lib/tasks/user.rake [hunk 1]
```diff
@@ -43,6 +43,29 @@ namespace :user do
...
+          user.save!
+          success "User role set to Administrator for email: #{email}"
+        else
+          info "Role 'Administrator' not found for provider 'greenlight'"
+        end
+      else
+        info "User with email: #{email} not found"
+      end
+    end
+  end
```
Comment (farhatahmad) [inline lib/tasks/user.rake:67]: The nested ifs make this super difficult to follow. Might be clear to use guard clauses here instead 

```
user = User.find_by(email:, provider: 'greenlight')
err 'User with email: #{email} not found' if user.blank?
...
```

Comment (SilentFlameCR) [reply lib/tasks/user.rake]: Yeah that sounds good. I updated the code with the guard clauses so its easier to read!

File: lib/tasks/user.rake [hunk 2]
```diff
@@ -43,6 +43,28 @@ namespace :user do
...
+    # return err if user not found
+    err "User with email: #{email} not found" if user.blank?
+
+    role = Role.find_by(name: 'Administrator', provider: 'greenlight')
+
+    # return err if Administrator role not found
+    err "Role 'Administrator' not found for provider 'greenlight'" if role.blank?
+
+    user.role = role
+    user.save!
```
Comment (farhatahmad) [inline lib/tasks/user.rake:64]: You can do this in one line `user.update(role:)`

Comment (SilentFlameCR) [reply lib/tasks/user.rake]: got it and updated!

**Review by farhatahmad — APPROVED** [review]

**Pull request merged**